src/probnum/filtsmooth/gaussfiltsmooth/kalmanposterior.py

```diff
@@ -122,3 +121,31 @@ def __len__(self):
 
     def __getitem__(self, idx):
         return self.state_rvs[idx]
+
+    def sample(self, locations=None, size=()):
```

Comment:
Why is `size=()` the default, and not `size=None` or `size=1`?

Reply:
because this is what we are doing in random variable inputs. Not sure why exactly, maybe `size=None` sends mixed signals (do you want to sample or not??).

Reply:
Ah I see, I didn't think of checking our other `sample` implementations. If it's done that way there, then it should be the same here, I agree :+1: